server/cardDealing/card.ts

```diff
@@ -0,0 +1,23 @@
+export class Card {
+  private cardId: number;
+  private color: string;
```

Comment:
ここunion型使うと結構いい感じになるやで(指定した文字列以外受け付けなくなるのでtypo心配なくなる && 入力補完使える)
```TypeScript
type Color = 'green' | 'purple' | 'yellow' | 'skullking' | 'pirates';
```
```suggestion
  private color: Color;
```